lib/runnerfactory.py

```diff
@@ -35,63 +36,52 @@ def getConfigNeutralizeOperatingSystem(config):
         except:
             pass
 
-        try:
-            appName = config.getValue(["appName"])
-            newConfig.setValue("appName", appName)
-        except:
-            pass
-
         newConfig.setValue("exe", executable)
 
         try:
             params = app.getValue(["params"])
             newConfig.setValue("params", params.copy())
         except:
             pass
+
+        for k in [["appName"], ["startup"]]:
+            if config.hasValue(k[0]):
+                newConfig.setValue(k[0], config.getValue(k))
+
         return newConfig
```

Comment:
I know it is very common practice to use short variables like this. I'd highly prefer the following.
```suggestion
        for key in [["appName"], ["startup"]]:
            if config.hasValue(key[0]):
                newConfig.setValue(key[0], config.getValue(key))

        return newConfig
```